src/cdk/activation.ts

```diff
@@ -76,23 +65,16 @@ function initializeIconPaths(context: vscode.ExtensionContext) {
 }
 
 async function registerCdkCommands(explorer: AwsCdkExplorer): Promise<void> {
-    registerCommand({
-        command: 'aws.cdk.provideFeedback',
-        callback: async () => {
-            vscode.env.openExternal(vscode.Uri.parse(cdkProvideFeedbackUrl))
-        },
-        telemetryName: 'Command_aws.cdk.provideFeedback'
+    vscode.commands.registerCommand('aws.cdk.provideFeedback', async () => {
+        vscode.env.openExternal(vscode.Uri.parse(cdkProvideFeedbackUrl))
+        recordCdkProvideFeedback()
     })
-    registerCommand({
-        command: 'aws.cdk.help',
-        callback: async () => {
-            vscode.env.openExternal(vscode.Uri.parse(cdkDocumentationUrl))
-        },
-        telemetryName: 'Command_aws.cdk.help'
+    vscode.commands.registerCommand('aws.cdk.help', async () => {
+        vscode.env.openExternal(vscode.Uri.parse(cdkDocumentationUrl))
+        recordCdkHelp()
     })
-    registerCommand({
-        command: 'aws.refreshCdkExplorer',
-        callback: async () => explorer.refresh(),
-        telemetryName: 'Command_aws.refreshCdkExplorer'
+    vscode.commands.registerCommand('aws.refreshCdkExplorer', async () => {
+        explorer.refresh()
```

Comment:
Metrics calls would be missed if function throws an Error

Reply:
same as above